src/x11/x.c

```diff
@@ -1030,7 +1098,51 @@ void x_setup(void)
         x_win_setup();
         x_cairo_setup();
         x_shortcut_grab(&settings.history_ks);
+}
+
+void x_win_round_corners(void)
```

Comment:
Please add a comment describing the method. Especially, **when** it should be called.